cfme/containers/image.py

```diff
@@ -39,6 +42,22 @@ def get_detail(self, *ident):
         navigate_to(self, 'Details')
         return InfoBlock.text(*ident)
 
+    def perform_smartstate_analysis(self, wait_for_finish=False):
+        """Performing SmartState Analysis on this Image
+        """
+        navigate_to(self, 'Details')
```

Comment:
We'd actually prefer to page the results instead of setting this, since it could take a very long time to load those 1000 images and the setting persists through navigation to another page and back.

Should be as simple as using paginator.pages() as you have in other spots in this PR, or using one of the PagedTable classes that has a 'find_row_by_cell_on_all_pages() method.